hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -22,12 +22,12 @@
     }
   ],
   wakeup_list: [
-    { name: "aon_gsc_wk",
+    { name: "aon_wk_req",
```

Comment:
super nit: not sure of `aon_` prefix is useful, given sysrst_ctrl is known to be in aon and all other wake up reqs to pwrmgr are also already in aon domain. Ditto below.

Reply:
this has to do with the `aon_` clock here, and not with the power domain.  i.e., inside `sysrst_ctrl` signals go back and forth between the AON clock and the bus clock domain, hence I have introduced these `aon_` prefixes as recommended per our style guide. does that make sense?

Reply:
I see, thanks for the explanation.